packages/core/admin/admin/src/translations/en.json

```diff
@@ -239,6 +239,8 @@
   "app.component.table.edit": "Edit {target}",
   "app.component.table.select.all-entries": "Select all entries",
   "app.component.table.select.one-entry": "Select {target}",
+  "app.components.menuBrand.title": "Strapi Dashboard",
```

Comment:
Can you rename them as `app.components.LeftMenu.navbrand.title` and ``app.components.LeftMenu.navbrand.workplace` ?

Reply:
Yes, that makes more sense.